package.json

```diff
@@ -26,7 +26,8 @@
         "onCommand:aws.invokeLambda",
         "onCommand:aws.deployLambda",
         "onCommand:aws.getLambdaConfig",
-        "onCommand:aws.getLambdaPolicy"
+        "onCommand:aws.getLambdaPolicy",
```

Comment:
Unconnected to this change but seeing it reminds me, I think we should remove the getLambdaConfig and getLambdaPolicy commands until we can eval if they are useful and if so, put a nicer UI around the output.

Reply:
Opened a corresponding issue here: #146